### PR TITLE
Simplifying implementation of is_safe.

### PR DESCRIPTION
**Pull request by C-Saunders**
This simplifies the implementation of `is_safe` and removes the iteration over the keys.

### All Submissions:

1. [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
2. [x] ~Have you written new tests for new features/fixes for bugs?~

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [ ] Have you linted your code locally prior to submission?
**What linter is in use? I don't see it in the README**

### Changes to Core Features:

1. [x] ~Have you written new tests?~
2. [ ] Have you linted your code locally prior to submission? **See above**
3. [x] Have you added an explanation to your changes and why you'd like them included?
4. [x] Does your submission pass tests locally?

### CODE REVIEW

**Comment by akaps** [comment]
I've been using pylint locally. Do you have a good recommendation other than that? My PR template largely was pulled from an example that covered stuff I liked

File: ai/player_utils.py [hunk 1]
```diff
@@ -14,11 +14,9 @@ def player_info(player_id, game_info):
     return game_info['known_info'][player_id]
 
 def is_safe(card, game_info):
+    [color, rank] = list(card)
```
Comment (akaps) [inline ai/player_utils.py]: Hell yea. Looking at the card rather than the tableau is the right way to do this. I can be a dummy sometimes

**Comment by C-Saunders** [comment]
> I've been using pylint locally. Do you have a good recommendation other than that?

Sorry, I totally missed this comment. I don't have a recommendation since I don't have any direct experience with any Python linters as far as I can remember :-/

`pylint` keeps complaining about not having a config file, so I guess I'd need yours? If you have one, it probably makes sense for it to be in the repo, right?

**Comment by akaps** [comment]
I've added issue #82 for having a designated linter. Will merge now

**Pull request merged**